packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/Resources/OpenshiftPlanResources.tsx

```diff
@@ -38,6 +38,22 @@ export const OpenshiftPlanResources: React.FC<{ planInventory: OpenshiftVM[] }>
   const missingCPUInfo = planInventory.find(({ object }) => getK8sCPU(object) === '0');
   const missingMemoryInfo = planInventory.find(({ object }) => getK8sVMMemory(object) === '0Mi');
 
+  const toDecAlignedFormat = (num) => {
```

Comment:
can you make this a reusable component ?

so we can user it like this:

`<AlignedDecimal value={123.5} unit={'MiB'} width={'lg'} />`